CMakeLists.txt

```diff
@@ -650,6 +650,10 @@ else()
 endif()
 
 find_package(OpenAL)
+if(OPENAL_FOUND)
```

Comment:
Currently FreeSurround only works on the OpenAL backend, this is why I made it look for OpenAL. There is no other reason.
It always use from externals, as I believe there will never be a system-wide FreeSurround library available.